example/index.android.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react'
+import {
+  AppRegistry,
+  StyleSheet,
+  Text,
+  View,
+  Button,
+} from 'react-native'
+import { TwitterModule } from 'tipsi-twitter'
+
+TwitterModule.init({
+  twitter_key: '<TWITTER_KEY>',
+  twitter_secret: '<TWITTER_SECRET>',
+})
+
+export default class example extends Component {
+  state = {
+    twitterUserId: '',
+    errorMessage: '',
+  }
+
+  handleCustomLoginPress = async () => {
+    try {
+      const result = await TwitterModule.logIn()
```

Comment:
Ok, but that's a moot point). https://ffeathers.wordpress.com/2014/04/06/login-or-log-in-a-way-to-choose-one-word-or-two/